### PR TITLE
Update docs for AppSec on Ruby 1.1.0

### PR DESCRIPTION
**Pull request by lloeki**
### What does this PR do?

Fixes some elements, and adds 1.1.0 features.

### Motivation

Upcoming ddtrace 1.1.0 for Ruby release.

### Additional Notes

Do not merge until 1.1.0 is released.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/application_security/setup_and_configure.md [hunk 1]
```diff
@@ -292,11 +292,51 @@ if span, ok := tracer.SpanFromContext(request.Context()); ok {
...
+  scope: 'read:message, write:files',
+
+  # optional free-form tags
+  another_tag: 'another_value',
+)
+```
+
+{{% /tab %}}
+
+{{% tab "Using trace `set_tag`" %}}
```
Comment (ivoanjo) [inline content/en/security_platform/application_security/setup_and_configure.md:331]: Less of a correction issue, but more of a documentation clarity issue: we seem to be giving users a lot of alternatives to do things: two ways of onboarding for each variant, two ways of setting tags, etc etc.

I wonder if we should be a bit more opinionated and just state one option? We could always serve advanced users using the API docs or some other advanced doc. See for instance <https://documentation.divio.com/introduction/#the-secret> for a discussion of this.

File: content/en/security_platform/application_security/getting_started/ruby.md [hunk 2]
```diff
@@ -71,15 +91,30 @@ You can monitor application security for Ruby apps running in Docker, Kubernetes
...
+     c.appsec.enabled = true
+     c.appsec.instrument :sinatra
+   end
+   ```
+
+   Or enable the APM tracer through auto-instrumentation:
+
+   ```ruby
+   require 'ddtrace'
+   require 'datadog/appsec/auto_instrument'
```
[the remark this answers is not in the document]
Comment (lloeki) [reply content/en/security_platform/application_security/getting_started/ruby.md:104]: Wait, no, that's a mistake, it's suppose to be `require 'ddtrace/auto_instrument'` 🤦

**Review by ivoanjo — APPROVED** [review]
👍 LGTM

File: content/en/security_platform/application_security/setup_and_configure.md [hunk 3]
```diff
@@ -292,11 +292,51 @@ if span, ok := tracer.SpanFromContext(request.Context()); ok {
...
+{{% tab "Using `Datadog::Kit::Identity.set_user` %}}
+
+Starting with `ddtrace` 1.1.0, a convenience `set_user` method is available:
+
+```ruby
+# Get the active trace
+trace = Datadog::Tracing.active_trace
+
+# Set mandatory user id tag
+Datadog::Kit::Identity.set_user(trace, id: 'd131dd02c56eeec4')
```
Comment (marcotc) [inline content/en/security_platform/application_security/setup_and_configure.md:308]: Not related to this PR, but wouldn't it make sense to mixin `Datadog::Kit::Identity` into `TraceOperation` to allow for `trace.set_user(trace, id: 'd131dd02c56eeec4')`?

**Review by marcotc — APPROVED** [review]

File: content/en/security_platform/application_security/setup_and_configure.md (hunk 3 above)
Comment (lloeki) [reply content/en/security_platform/application_security/setup_and_configure.md:308]: That might be an option users have to do manually. I'm not sure we want that automatically.

**Comment by kayayarai** [comment]
Edit pass complete. Let me know if it looks alright to you. Previews available here: 

https://docs-staging.datadoghq.com/kari/update-appsec-docs-for-ruby-1-1/security_platform/application_security/getting_started/ruby/
https://docs-staging.datadoghq.com/kari/update-appsec-docs-for-ruby-1-1/security_platform/application_security/setup_and_configure/

(Previews are generated only for branches with a `/` character in them)

**Pull request merged**